core/build.sbt

```diff
@@ -117,6 +117,8 @@ lazy val streamlets =
             Ficus,
             Avro,
             Bijection,
+            LogbackClassic,
```

Comment:
Would prefer if we don't do this, and let users add logback, and we tell them they should.